### ISSUE TITLE
ansible(slicer): Fix install of extensions

### ISSUE THREAD

**Issue by jcfr** [MorphoCloud/MorphoCloudWorkflow#2]
From @muratmaga 

> Extensions are not installed by default.

**Comment by jcfr** [comment]
```
slicer_extensions:
  - SlicerMorph
  - SurfaceWrapSolidify
```

Source: https://github.com/jcfr/exosphere/blob/5f85d93d58c746856019dce43c8686e1eeadcb36/ansible/roles/slicer/vars/main.yml#L14-L16

```
- name: 'Slicer extensions installed'
  become: yes
  become_user: exouser
  command: "{{ install_dir }}/dist/slicer-download.sh -d {{ install_dir }}/Slicer -v {{ slicer_release_id }} -e {{ item }}"
  loop: '{{ slicer_extensions }}'
```

Source: https://github.com/jcfr/exosphere/blob/5f85d93d58c746856019dce43c8686e1eeadcb36/ansible/roles/slicer/tasks/main.yml#L82-L86

**Comment by jcfr** [comment]
Extensions are indeed installed:

| SlicerMorph | SurfaceWrapSolidify |
|--|--|
| ![image](https://github.com/user-attachments/assets/410493ee-9baf-4c98-8645-be6cbd49fa06) | ![image](https://github.com/user-attachments/assets/31ef5d40-a5cb-458a-915a-053c183f1dd5) |


Issue is that extension manager settings are not current.

**Comment by jcfr** [comment]
Fixed in MorphoCloud/exosphere@57c952b1 and integrated through https://github.com/MorphoCloud/MorphoCloudWorkflow/commit/2c82cc300f958a95be5ec992a31677034b2ee721
